docs/source/debugger/expressions.rst

```diff
@@ -30,15 +30,19 @@ Differences from C Behaviors
 ----------------------------
 
 
-- First, all math is performed on full 64-bit unsigned values, so things like a < 0 won't work as expected. 
+- First, all math is performed on full 64-bit unsigned values, so things like **a < 0** won't work as expected. 
 
-- Second, the logical operators && and || do not have short-circuit properties -- both halves are always evaluated. 
+- Second, the logical operators **&&** and **||** do not have short-circuit properties -- both halves are always evaluated. 
 
 - Finally, the new memory operators work like this:
 
-  - b@<addr> refers to the byte read from <addr>. 
+  - **b!<addr>** refers to the byte read from <addr> but does *NOT* suppress side effects such as reading a mailbox clearing the pending flag, or reading a FIFO removing an item.
+  
+  - **b@<addr>** refers to the byte read from <addr> while suppressing side effects. 
```

Comment:
This should say "the byte at" in both cases - it's applicable for writing as well as reading.